test/runtests.jl

```diff
@@ -74,5 +74,9 @@ include("extensions/enzyme.jl")
 @static if VERSION >= v"1.7.0"
     @testset "Enzyme" begin
         enzyme_testsuite(CPU, Array)
+        # Requires CUDA
+        # if CUDA.functional() && CUDA.has_cuda_gpu()
```

Comment:
This should probably be enabled?

Reply:
In a previous comment, you mentioned that CUDA should not be part of the basic tests run on GitHub Actions. The test is enabled on buildkite.

Reply:
Then we should delete those lines since they are noise, but we should still add some more complicated kernels, but I guess we can do that in a later PR (famous last words.)